src/app/domains/wallets/domains.wallets.gateway.ts

```diff
@@ -14,18 +16,24 @@ export class DomainsWalletsGateway {
   }
 
   public getWallets(): Observable<IWalletApiResponse[]> {
-    return this.http.get<IWalletApiResponse[]>(API_WALLETS_URL);
+    return fakeRequest(GET_WALLETS_API_RESPONSE_MOCK());
+    //return this.http.get<IWalletApiResponse[]>(API_WALLETS_URL); // TE ZMIANY SA CHWILOWE, BO MI DALEJ BACKEND NIE DZIALA
   }
 
   public createWallet({ name }: IWalletPayload): Observable<IWalletApiResponse> {
     return this.http.post<IWalletApiResponse>(API_WALLETS_URL, { name });
   }
 
   public updateWallet(id: number, { name }: IWalletPayload): Observable<IWalletApiResponse> {
-    return this.http.put<IWalletApiResponse>(API_WALLETS_URL, { id, name });
+    return fakeRequest({id: 1, creationDate: new Date().toString(), name})
+    return this.http.put<IWalletApiResponse>(API_WALLETS_URL, { id, name }); // TE ZMIANY SA CHWILOWE, BO MI DALEJ BACKEND NIE DZIALA
   }
 
   public deleteWallet(id: number): Observable<void> {
     return this.http.delete<void>(API_WALLETS_URL + `/${id}`);
   }
+
+  public getWalletsDetails(id: number): Observable<IWalletDetailsApiResponse> {
```

Comment:
Can be removed, there is not such endpoint planned for now. Get wallet details from the wallets list request by finding wallet by ID.

Reply:
Was not done yet